### ISSUE TITLE
Getting into software engineering, and other anecdotes

### ISSUE THREAD

**Issue by bouwe77** [bouwe77/curious-coders#17]
> Just thinking out loud here 🤔 

I think both of us can tell a lot about what made us decide going into coding, IT, engineering, whatever you wanna call it.

For example, early childhood memories and things you did, which, looking back, are things why it's not surprising you ended up being an engineer.

Or what was it what you drew into coding, and which of these things are still driving you, and which things are new, that you discovered later?

etc...

**Comment by RichStone** [comment]
Yeah man, awesome questions. Also, maybe we can talk here about one of me mysterious questions, why so many musicians become coders :))

**Comment by bouwe77** [comment]
Ah yes indeed, remarkable!
